docs/enterprise-setup/implementation-guide.md

```diff
@@ -230,32 +229,23 @@ minio:
   enabled: false
 
 global:
-    log4jConfig: "log4j2-no-minio.xml"
-    logs:
-        storage:
-            type: "S3"
-        
-        minio:
-            enabled: false
-
-        s3:
-            enabled: true
-            bucket: "" ## S3 bucket name that you've created.
-            bucketRegion: "" ## e.g. us-east-1
-
-        accessKey: ## AWS Access Key.
-            password: ""
-            existingSecret: "" ## The name of an existing Kubernetes secret containing the AWS Access Key.
-            existingSecretKey: "" ## The Kubernetes secret key containing the AWS Access Key.
-
-        secretKey: ## AWS Secret Access Key
-            password:
-            existingSecret: "" ## The name of an existing Kubernetes secret containing the AWS Secret Access Key.
-            existingSecretKey: "" ## The name of an existing Kubernetes secret containing the AWS Secret Access Key.
+  storage:
+    type: "S3"
+    bucket: ## S3 bucket names that you've created. We recommend storing the following all in one bucket.
+      log: airbyte-bucket
+      state: airbyte-bucket
+      workloadOutput: airbyte-bucket
+
+    s3:
+      region: "" ## e.g. us-east-1
+      accessKey: ## AWS Access Key.
+        accessKeyExistingSecret: ## The name of an existing Kubernetes secret containing the AWS Access Key.
+        accessKeyExistingSecretKey: ## The Kubernetes secret key containing the AWS Access Key.
+      secretKey: ## AWS Secret Access Key
+        secretKeyExistingSecret: ## The name of an existing Kubernetes secret containing the AWS Secret Access Key.
+        secretKeyExistingSecretKey: ## The name of an existing Kubernetes secret containing the AWS Secret Access Key.
```

Comment:
The `existingSecret` values should not be indented.
```yaml
      accessKey: ## AWS Access Key.
      accessKeyExistingSecret: ## The name of an existing Kubernetes secret containing the AWS Access Key.
      accessKeyExistingSecretKey: ## The Kubernetes secret key containing the AWS Access Key.
      
      secretKey: ## AWS Secret Access Key
      secretKeyExistingSecret: ## The name of an existing Kubernetes secret containing the AWS Secret Access Key.
      secretKeyExistingSecretKey: ## The name of an existing Kubernetes secret containing the AWS Secret Access Key.
```

Reply:
What do you fill in for `accessKey` and `secretKey` then?